siliconcompiler/core.py

```diff
@@ -1930,7 +1930,7 @@ def write_manifest(self, filename, prune=True, abspath=False):
 
         Args:
             filename (filepath): Output filepath
-            prune (bool): If True, essential non-empty parameters from the
+            prune (bool): If True, only essential parameters from the
```

Comment:
Maybe just say "only essential fields" (or just specify that "help" and "example" are removed) now that it's not actually deleting any parameters.